### PR TITLE
Issue/245 image in paragraph

### LINKED ISSUE #245: Image inside a paragraph breaks the HTML
1. Start with `<p>hello</p>` in HTML mode
2. Switch to visual mode
3. Add an image
4. Notice the text is modified to `hell` and the and `>` is added after the image

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #245.

Can someone have a quick look so that it can be included in the release? Thanks!

### CODE REVIEW

File: aztec/src/main/java/org/wordpress/aztec/Html.java [hunk 1]
```diff
@@ -751,28 +751,6 @@ private static void end(SpannableStringBuilder text, TextFormat textFormat) {
         }
     }
 
-//<<<<<<< HEAD
```
Comment (theck13) [inline aztec/src/main/java/org/wordpress/aztec/Html.java]: Good idea.  We can remove [this unused code](https://github.com/wordpress-mobile/WordPress-Aztec-Android/pull/246/files#diff-7177c376b871e9e1aa2e4983072f0705L164) too.

File: aztec/src/main/kotlin/org/wordpress/aztec/formatting/LineBlockFormatter.kt [hunk 2]
```diff
@@ -273,32 +270,31 @@ class LineBlockFormatter(editor: AztecText, val headerStyle: LineBlockFormatter.
 
     fun insertMedia(drawable: Drawable?, attributes: Attributes, onMediaTappedListener: OnMediaTappedListener?) {
         val span = AztecMediaSpan(editor.context, drawable, attributes, onMediaTappedListener)
-        val html = span.getHtml();
 
-        val isAddingMediaToEndOfBlockElement = editableText.getSpans(selectionStart, selectionEnd, AztecBlockSpan::class.java)
-                .any {
-                    selectionStart == editableText.getSpanEnd(it)
-                }
+        val spanAtEndOfWhichMediaIsAdded = editableText.getSpans(selectionStart, selectionEnd, AztecBlockSpan::class.java)
```
Comment (theck13) [inline aztec/src/main/kotlin/org/wordpress/aztec/formatting/LineBlockFormatter.kt]: It looks like `spanAtEndOfWhichMediaIsAdded` was renamed similar to `spanAtStartOfWhichMediaIsAdded`.  What about naming them `spanBeforeMedia` and `spanAfterMedia`, respectively?  That's easier to grok.

Comment (0nko) [inline aztec/src/main/kotlin/org/wordpress/aztec/formatting/LineBlockFormatter.kt]: Oh yeah :) makes sense..

File: aztec/src/main/java/org/wordpress/aztec/Html.java (hunk 1 above)
Comment (0nko) [inline aztec/src/main/java/org/wordpress/aztec/Html.java]: Good catch, time to get rid of it.

**Comment by 0nko** [comment]
Thanks for the quick review @theck13.

**Pull request merged**